vendor.conf

```diff
@@ -79,7 +79,7 @@ google.golang.org/grpc v1.12.0
 # the containerd project first, and update both after that is merged.
 # This commit does not need to match RUNC_COMMIT as it is used for helper
 # packages but should be newer or equal.
-github.com/opencontainers/runc 96ec2177ae841256168fcf76954f7177af9446eb
+github.com/opencontainers/runc 12f6a991201fdb8f82579582d5e00e28fba06d0a
```

Comment:
Should we wait for the containerd version to be bumped first? (see the comment above); (assuming this will be a containerd 1.2.3 release)

Reply:
No change on containerd code: https://github.com/containerd/containerd/pull/2932

Reply:
right, but the tricky it is that (given `runc` hasn't reached 1.0 yet, so doesn't have separate packages in the docker repository), the `containerd.io` package (afaik) uses the version that's in the containerd tag, so if we want those packages to be updated, it needs a new tag in containerd with the version of `runc` bumped 😞

Reply:
Actually; changes are only in libcontainer, correct, so the `runc` _binary_ does not have to be updated (?)

Reply:
Oh, never mind; likely `runc` binary uses `libcontainer` as well.. sorry.. my brain went AWOL for a bit